core/common/src/main/kotlin/de/tum/informatics/www1/artemis/native_app/core/common/markdown/PostArtemisMarkdownTransformer.kt

```diff
@@ -1,26 +1,41 @@
 package de.tum.informatics.www1.artemis.native_app.core.common.markdown
 
+import de.tum.informatics.www1.artemis.native_app.core.common.R
+
 class PostArtemisMarkdownTransformer(val serverUrl: String, val courseId: Long) : ArtemisMarkdownTransformer() {
-    override fun transformExerciseMarkdown(title: String, url: String): String {
-        return "[$title](artemis:/$url)"
+
+    private val resourcePath = "android.resource://de.tum.cit.aet.artemis/"
```

Comment:
Make this a public constant that we can then also use in `MarkdownText.createMarkdownRender()`

Reply:
Good idea :)